lib/generators/thredded/install/templates/initializer.rb

```diff
@@ -21,6 +21,18 @@
 # User avatar URL. rb-gravatar gem is used by default:
 Thredded.avatar_url = ->(user) { Gravatar.src(user.email, 128, 'mm') }
 
+# ==> Permissions Configuration
+# By default, thredded uses a simple permission model, where all the users can post to all message boards,
+# and admins and moderators are determined by a flag on the users table.
+
+# The name of the moderator flag column on the users table.
+Thredded.moderator_column = :admin
+# The name of the admin flag column on the users table.
+Thredded.admin_column = :admin
```

Comment:
I'm assuming the columns here are booleans - is that correct? Oh, looks like it. Just saw it here `20160308104204_add_admin_to_users.rb`. I'd recommend updating the comments above to say something like:

`# The name of the moderator flag column, a boolean column, on the users table.`
and
`# The name of the admin flag column, a boolean column, on the users table.`